pom.xml

```diff
@@ -612,6 +614,12 @@
     <!-- By default only check remote repositories once per week -->
     <maven.repository.update.freqency>interval:10080</maven.repository.update.freqency>
 
+    <!-- Starting from Jenkins 1.625.x Jenkins supports Java 7 only, hence we may want to bump the dependency in components -->
+    <java.level>6</java.level>
```

Comment:
Then we need to bump a major version number. Otherwise people may upgrade and get side effects in their projects.